autopush_rs/src/server/mod.rs

```diff
@@ -338,9 +342,19 @@ impl Server {
         } else {
             ServiceChangeTracker::new(Vec::new())
         };
+        let region = if env::var("AWS_PROFILE").unwrap_or("default".to_string()) == "test" {
```

Comment:
use `if cfg!(test)` here?

Reply:
Do we set what ever that checks somewhere?

Reply:
`cfg!()` can be called at run time to see what configuration you happen to be in. It's the inline equivalent of "#[cfg(test)]". 

It might be more useful than checking the AWS_PROFILE for a test configuration value.

Reply:
I mean, how do we flag in the environment that its in test mode? It's not running a Rust based test suite that would flag this afaik.

Reply:
Right, the #cfg test toggle is via a compiler flag, so unfortunately we can't use it here.

The python code keys on AWS_LOCAL_DYNAMODB, so I'd rather this key on that as well. Then I don't think the changes to .travis.yml/boto.cfg/tox.ini are necessary (and local testing should continue to work as normal)